observations/README.md

```diff
@@ -0,0 +1,1279 @@
+Blood Pressure Observations are mapped to a `CompoundStatement` with `ObservationStatement` components for the systolic 
+and diastolic readings and a optional `NarrativeStatement` for additional information.
+
+### Parent Compound Statement
+
+| Mapped to (XML HL7 CompoundStatement) | Mapped from (JSON FHIR / other source )                                                                                                                      |
+|---------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id                                    | Unique ID generated by the adaptor                                                                                                                           |
+| code \[@code]                         | `Observation.code.coding[0].code`                                                                                                                            |
+| code \[@displayName]                  | `Observation.code.coding[0].display`                                                                                                                         |
+| code \[@codeSystem]                   | fixed value = `"2.16.840.1.113883.2.1.3.2.4.15"`                                                                                                             |
+| code / originalText                   | `coding.text` <br/> or else `coding.display` <br/> or else `coding.extension[index].value` where `coding.extension[index].url` equals `"descriptionDisplay"` |
+| statusCode                            | fixed value = `"COMPLETE"`                                                                                                                                   |
+| effectiveTime                         | `Observation.effectiveDateTime` or else `Observation.effectivePeriod`                                                                                        |
+| availabilityTime \[@value]            | `Observation.effectiveDateTime` or else `Observation.effectivePeriod.start`                                                                                  |
+| Participant \[@typecode]              | fixed value = `"PRF"`                                                                                                                                        |
+| Participant / agentRef / id \[@root]  | `Observation.performer[0]`                                                                                                                                   | 
+
+### Observation components
+
+The blood pressure should have two `ObservationStatement` components, one with a systolic and one with a diastolic 
```

Comment:
I find the word `should` confusing here.  Given that we're (the software) is generating the `ObservationStatement` should it be `will`? If `should` is correct, should we be clear about what situations there won't be two?

Reply:
I've added clarification and added another link to the documentation, which explains it better. Also added  "If the blood pressure isn't in the triple structure it will be mapped as an uncategorised observation" to the section above.